### PR TITLE
cleanzzzzz

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/strings-ii/using-test-with-regexp.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/strings-ii/using-test-with-regexp.md?ref=384b25aae40d45b23bd62f4170fe4f136fbaab07)

File: javascript/core/strings-ii/using-test-with-regexp.md [hunk 1]
```diff
@@ -1,45 +1,17 @@
...
-  - html
-
-  - introduction
-
-  - workout
-
-
 links:
-
   - '[developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test){website}'
```
Comment (kapnobatai137) [inline javascript/core/strings-ii/using-test-with-regexp.md]:
```suggestion
  - '[Official Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test){documentation}'
```

File: javascript/core/strings-ii/using-test-with-regexp.md [hunk 2]
```diff
@@ -1,45 +1,17 @@
...
 
-
 aspects:
   - introduction
   - workout
 
-
 ---
 
 # Using `.test()` with RegExp
```
Comment (kapnobatai137) [inline javascript/core/strings-ii/using-test-with-regexp.md]:
```suggestion
# Using `.test()` With RegExp
```

**Comment by kapnobatai137** [comment]
will check whether we explain what RegEx is in this/previous workout. if not, will add a better explanation

**Pull request merged**